homeassistant/components/shelly/switch.py

```diff
@@ -43,6 +64,16 @@ def async_setup_block_entry(
     coordinator = get_entry_data(hass)[config_entry.entry_id].block
     assert coordinator
 
+    # Add Shelly Gas Valve as a switch
+    if coordinator.model == "SHGS-1":
+        async_setup_block_attribute_entities(
+            hass,
+            async_add_entities,
+            coordinator,
+            {("valve", "valve"): GAS_VALVE_SWITCH},
+            BlockValveSwitch,
+        )
```

Comment:
Should we add a return here? Do we expect relay blocks in this device?

Reply:
Good point, there is no relay block for Shelly Gas. Thanks.